### PR TITLE
Added violation for multiline loops

### LINKED ISSUE #813: Forbid multi-line for and while loops
# Rule request

## Thesis

Code [like this](
https://github.com/sobolevn/itmo-2019/pull/24/files#diff-5ad37df7dcdec8d4aeb51aac0bde8387R51-R54) should be forbidden: 

```python
    for test_file in find_files(
        target_dir,
        '{0}*{1}'.format(TEST_PREFIX, PY_EXTENSION),
    ):
        module = import_module(test_file)
```

## Reasoning

`for`s should be simple and readable. The correct way to rewrite the same code is:

```python
    files = find_files(
        target_dir,
        '{0}*{1}'.format(TEST_PREFIX, PY_EXTENSION),
    )
    for test_file in files:
        module = import_module(test_file)
```

This is a consistency rule for `ast.For`, `ast.AsyncFor`, and `ast.While` nodes.

### PR DESCRIPTION
**Pull request by Ben-Drebing**
# I have made things!
Implemented the `MultiLineLoopViolation` which is produced when the "iterator" section of a for-loop and the "test" section of a while-loop take up multiple lines.

## Checklist

- [X] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [X] I have created at least one test case for the changes I have made
- [X] I have updated the documentation for the changes I have made
- [X] I have added my changes to the `CHANGELOG.md`

## Related issues
Closes #813

### CODE REVIEW

File: wemake_python_styleguide/violations/consistency.py [hunk 1]
```diff
@@ -1838,3 +1840,31 @@ class RedundantSubscriptViolation(ASTViolation):
...
+class MultilineLoopViolation(ASTViolation):
+    """
+    Forbids multiline loops.
+
+    Reasoning:
+        Decreased the readability of the code.
+
+    Example::
+
+        #Correct
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/consistency.py]: `# Correct:`

File: wemake_python_styleguide/violations/consistency.py [hunk 2]
```diff
@@ -1838,3 +1840,31 @@ class RedundantSubscriptViolation(ASTViolation):
...
+
+        for num in SomeFunc(arg1, arg2):
+            ...
+
+        #Wrong
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/consistency.py]: `# Wrong:`

**Comment by Ben-Drebing** [comment]
Should be all good to go now! You might have noticed I remembered to run flake8 this time :)

**Comment by sobolevn** [comment]
@Ben-Drebing can you please fix this issue? 

```
$ make test
mypy wemake_python_styleguide
wemake_python_styleguide/visitors/ast/loops.py:196: error: Argument 1 to "add_violation" of "BaseVisitor" has incompatible type "Type[MultilineLoopViolation]"; expected "BaseViolation"
Found 1 error in 1 file (checked 102 source files)
```

**Pull request merged**